cluster/saltbase/salt/fluentd-gcp/fluentd-gcp.yaml

```diff
@@ -11,7 +11,11 @@ spec:
   dnsPolicy: Default
   containers:
   - name: fluentd-cloud-logging
-    image: gcr.io/google_containers/fluentd-gcp:1.29
+    image: gcr.io/google_containers/fluentd-gcp:1.30
+    command:
+      - '/bin/sh'
+      - '-c'
+      - '/run.sh $FLUENTD_ARGS 2>&1 >>/var/log/fluentd.log'
```

Comment:
But no args are being passed here, so we can expect there is no args. Did I miss something? Once they will be introduced you can add them either by expanding the command or introducing them as args